android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -515,18 +511,6 @@ private int getDownloadMapsCounter()
     return info == null ? 0 : info.filesCount;
   }
 
-  @Override
-  public void onNoConnectionError()
-  {
-    dismissAlertDialog();
-    mAlertDialog = new MaterialAlertDialogBuilder(this, R.style.MwmTheme_AlertDialog)
-        .setTitle(R.string.common_check_internet_connection_dialog_title)
```

Comment:
Was this functionality broken/lost during refactorings? Why is it removed? Maybe it should be restored instead. What was it about compared to 2020 mapsme version?

Reply:
The only one caller of `onNoConnectionError()` was AbstractGuidesClickListener:

https://github.com/mapsme/omim/blob/1892903b63f2c85b16ed4966d21fe76aba06b9ba/android/src/com/mapswithme/maps/maplayer/guides/AbstractGuidesClickListener.java#L27-L38